ironfish/src/wallet/account/encoder/multiSigKeys.ts

```diff
@@ -64,7 +64,8 @@ export function isSignerMultiSig(multiSigKeys: MultiSigKeys): multiSigKeys is Mu
 }
 
 export function AssertIsSignerMultiSig(
```

Comment:
We should have a separate assertion for `AssertIsMultiSig` vs `AssertIsSignerMultiSig`, this change should be reverted.